### ISSUE TITLE
Saving macrostates directly to PDB

### ISSUE THREAD

**Issue by andt88** [Acellera/htmd#797]
Hi all,
is it possible to save the structures generated by viewStates directly to e.g. PDB without opening VMD?
Thanks a lot for your help!

**Comment by stefdoerr** [comment]
Yes, use `getStates` instead which retults a list of `Molecule` objects. Take into account that `Molecule.write('x.pdb')` does not write out all frames, just the current one. So you will need to loop over the frames

```python
res = model.getStates()
for i, stmol in enumerate(res):
   for f in range(stmol.numFrames):
      stmol.frame = f
      stmol.write('state{}_frame{}.pdb'.format(i, f))
```
Something like this should do it (I didn't test it)

**Comment by andt88** [comment]
Thanks a lot, I am trying it right now!
